src/server.conf.in

```diff
@@ -157,10 +157,10 @@ enabled = false
 ;   when allowRegistration is true. While certificate mode offers the highest
 ;   security it is currently considered experimental and the user experience
 ;   varies between browsers and platforms.
-;mode = sharedsecret
+mode = sharedsecret
 ; The shared secred for HMAC validation in "sharedsecret" mode. Best use 32 or
 ; 64 bytes of random data.
-;sharedsecret_secret = some-secret-do-not-keep
+sharedsecret_secret = some-secret-do-not-keep
```

Comment:
Users have to manually change that in server.conf and restart the service. I've documented that process in the readme.
Since the secret in Nextcloud is always going to be different from this default value, I don't think there is going to be a problem, but correct me if I'm wrong :)

Now obviously, I would love for the value to be automatically populated in server.conf... but I'm not sure how to do this without creating a special version of the plugin which communicates with the spreedme snap...
I don't think it would be wise to save it to disk in the app's folder.

Any other ideas?